### PR TITLE
CIV-10493 Record Judgment UI

### PR DESCRIPTION
**Pull request by hmcts-version1-ignacio**
### As a Caseworker I want to be able to record a Judgment following a Judge's order or Determination of Means So that if the Claimant wishes to enforce the claim, the Judgment can be registered with RTL ###

image

image

image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/CaseField/CaseField-JO-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    "CaseTypeID": "CIVIL",
+    "ID": "joIsLiveJudgmentExists",
+    "Label": " ",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "judgmentDetails",
```
Comment (Azam-Hmcts) [inline ccd-definition/CaseField/CaseField-JO-nonprod.json:20]: Pleas modify name of variable which should indicate that its label. We can add lblJudgementDetails.

File: ccd-definition/FixedLists/JudgmentOnline/RePaymentPlanSelection-JO-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+[
```
Comment (Azam-Hmcts) [inline ccd-definition/FixedLists/JudgmentOnline/RePaymentPlanSelection-JO-nonprod.json:1]: I would suggest to change name of file and remove the word "Re".

File: ccd-definition/CaseEventToComplexTypes/JudgmentStatusDetails-JO-nonprod.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "ID": "JudgmentStatusDetailsList",
```
Comment (Azam-Hmcts) [inline ccd-definition/CaseEventToComplexTypes/JudgmentStatusDetails-JO-nonprod.json:3]: I think we can rename all ID of types in this fiel to "JudgmentStatusDetails" like casefield ID. It contains one field of "judgmentStatusTypes" which itself is type of "JudgmentStatusDetailsList". So the current ID can be bit confusing.

File: ccd-definition/CaseField/CaseField-JO-nonprod.json (hunk 1 above)
Comment (hmcts-version1-ignacio) [reply ccd-definition/CaseField/CaseField-JO-nonprod.json:20]: It is the micro text displaying "Record Judgment" on top so I changed it to: lblRecordJudgment

File: ccd-definition/CaseField/CaseField-JO-nonprod.json [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    "ID": "joIsRegisteredWithRTL",
+    "Label": "Should this judgment be registered with RTL?",
+    "HintText": "This judgment should only be registered with RTL if the judge has ordered the amount should be paid in instalments, or if the claimant has applied to enforce the judgment",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "joSameJudgment",
```
[the remark this answers is not in the document]
Comment (hmcts-version1-ignacio) [reply ccd-definition/CaseField/CaseField-JO-nonprod.json:106]: True, changed to the suggested.

File: ccd-definition/CaseEventToComplexTypes/JudgmentStatusDetails-JO-nonprod.json (hunk 3 above)
Comment (hmcts-version1-ignacio) [reply ccd-definition/CaseEventToComplexTypes/JudgmentStatusDetails-JO-nonprod.json]: Renamed!

**Review by bhagyashreesharma90 — APPROVED** [review]

**Review by Azam-Hmcts — APPROVED** [review]

**Review by ShwetaTandel-hmcts — APPROVED** [review]

**Review by sampankumar — APPROVED** [review]

**Pull request merged**